cache-filter/src/filter/root.rs

```diff
@@ -83,6 +96,90 @@ impl RootContext for CacheFilterRoot {
         }
     }
 
+    fn on_queue_ready(&mut self, queue_id: u32) {
+        info!(
+            self.context_id,
+            "thread({}): on_queue called on the filter side", self.id
+        );
+        match self.dequeue_shared_queue(queue_id) {
+            Ok(Some(bytes)) => {
+                let ctxt_str = std::str::from_utf8(&bytes).unwrap();
+                let context_to_resume: u32 = match ctxt_str.parse() {
+                    Ok(ctxt_id) => ctxt_id,
+                    Err(e) => {
+                        warn!(
+                            self.context_id,
+                            "failed to parse message({}) from MQ into ctxt id: {}", ctxt_str, e
+                        );
+                        return;
+                    }
+                };
+
+                WAITING_CONTEXTS.with(|refcell| {
```

Comment:
After resuming the http request, `context` stored in `WAITING_CONTEXTS` should be removed.

Reply:
Fixed